### ISSUE TITLE
Wordpress container (e.g. wp_1) does not run in Windows 10 Desktop Docker

### ISSUE THREAD

**Issue by palomnik** [nezhar/wordpress-docker-compose#42]
This Wordpress-Docker-Compose setup runs brilliantly on MacOSX and Linux for me, but I cannot get it to run on WIndows 10 (Docker Desktop).  I have tried running in powershell as user or as administrator, but in each case, the Wordpress Container is not running.  I could not find anything in the logs.  I can get to phpmyadmin, and the DB is created but not populated.

I'm not sure where to start looking for the problem.

**Comment by palomnik** [comment]
Using Portainer helped me track down the problem of which there were two:
Files in wp-app could not load.  I thought it was perms, but it was my stupidity: With Dropbox Smart-Sync, the files appear to be there, even if they aren't - I had designated the files to be "oneline only" - duh!  Downloading the files solved the first problem.

The second problem was that the database would not load.  I usually solve this by just re-importing the SQL file via PMA.  But the SQL file was 4M, and PMA would only accept 2M or less (apparently the added php config did not solve this for PMA - I'll have to figure this out later). I could have sshed to the DB container, but Portainer makes it easy to get a prompt, so I used that instead, and just used mysql to import the sql file. 

In summary, I now have sets of folders in which the docker-compose from here reside, and the created files - in Dropbox.  I am able to bring up all of my sites in Linux, on Mac and on Windows, as long as I am careful to let Dropbox finish syncing, and I must occasionally check and fix permissions and reimport the latest saved database.  I added mailhog to the docker-compose.yml, and create a plugin to send mail there.

Now I just have to resolve some problems with wp-cli.

**Comment by palomnik** [comment]
Oh, and while it is nice to have Windows as a backup possibility, Linux and Mac are definitely my preferred wordpress development OSes.
